portfolio/src/main/webapp/index.html

```diff
@@ -206,9 +210,28 @@ <h5>Mount Diablo</h5>
       </div>
     </div>
 
-    <!-- Bootstrap JS -->
+    <!-- Footer -->
+    <div id="footer-container">/
+        <a href="https://linkedin.com/in/ayakkala/">
+          <img class="logo-image" src="images/linkedIn-logo.png">
```

Comment:
Add alt text to all images

Reply:
I think Li mentioned this in a previous pull request. It should be fixed once I merge with master.